#!/usr/bin/env python3
PACKAGE = "wall_follow"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kp", double_t, 0, "kp",    14, 0,   20)
gen.add("kd", double_t, 0, "kd",    .09, 0,   1)
gen.add("ki", double_t, 0, "ki",    0, 0,   1)
gen.add("fast_speed", double_t, 0, "fast_speed", 0.75, 0, 10)
gen.add("medium_speed", double_t, 0, "medium_speed", 0.5, 0, 10)
gen.add("slow_speed", double_t, 0, "slow_speed", 0.25, 0, 10)


exit(gen.generate(PACKAGE, "wall_follow", "wall_follow_dynamic"))
